src/Components/PetList.js

```diff
@@ -1,9 +1,39 @@
-export const PetList = () => {
-  return (
-    <aside className="pets-list">
-      <p>No pets listed for this employee.</p>
-    </aside>
-  );
+import React, { useState, useEffect } from "react";
+
+export const PetList = ({ id, toggle, setToggle, cid }) => {
+  const [currPets, setPets] = useState([]);
+
+  useEffect(() => {
+    fetch("https://vet-resource-api-9-2.herokuapp.com/api/pets", {
```

Comment:
Could've also used the /api/pets?employeeId={id} URL param to have the server do the filtering by pets for you